XLS-0045-prepublish-validator-lists/README.md

```diff
@@ -0,0 +1,824 @@
+<pre>
+title: <b>Prepublish Validator Lists</b>
+type: released in rippled 1.7.0
+description: Future activation date and improved expiration handling of UNLs
+author: <a href="mailto:ed@ripple.com">Ed Hennis (Ripple)</a>
+</pre>
+
+
+## Abstract
+
+Adds a future effective date to UNLs so that all nodes in the peer network can switch to a new UNL simultaneously. 
+Creates a new version of the VL file spec to allow publishing multiple UNLs in one file, including current and
+future UNLs. Adds enforcement of the UNL expiration date so that rippled will act similarly to being amendment
+blocked if it has no active valid UNL.
```

Comment:
```suggestion
Each XRPL server (node) must be configured with a UNL (Unique Node List), which
lists the validating nodes that the server considers trusted. A validator list
includes the public key of the list publisher, a manifest, a signature, a
version, and a base64-encoded blob that contains a UNL.

This document defines version 2 of the VL (Validator List) spec. Compared to
version 1, it adds a future effective date so that nodes in the peer-to-peer
network can switch to a new UNL simultaneously. It also supports publishing
multiple UNLs in one file, allowing future UNLs to be included alongside a
current UNL.

This spec also defines the enforcement of the UNL expiration date so that a
server will act similarly to being amendment-blocked if it has no active valid
UNL.
```
(hard wrapped to 80)